pandas/core/frame.py

```diff
@@ -6181,6 +6181,15 @@ def dropna(
              name        toy       born
         1  Batman  Batmobile 1940-04-25
         """
+        if (how is not None) and (thresh is not None):
+            raise ValueError(
+                "You cannot set both the how and thresh arguments at the same time."
+            )
+
+        # default value for how
+        if how is None:
+            how = "any"
```

Comment:
perhaps replace this with
```suggestion
    how = how or "any"
```

Reply:
OK, let's use `lib.no_default` and remove the `default value` comment